### ISSUE TITLE
Unintended removal of backslashes from file paths inside JSON strings

### ISSUE THREAD

**Issue by horatiohafnaugels** [goatfungus/NMSSaveEditor#165]
Comparing a save file content before and after editing with NMSSaveEditor, I noticed paths like this:
`"MODELS\/COMMON\/WEAPONS\/MULTITOOL\/MULTITOOL.SCENE.MBIN"`
get automatically converted to paths like this:
`"MODELS/COMMON/WEAPONS/MULTITOOL/MULTITOOL.SCENE.MBIN"`

The same thing happens with IDs like this:
`"tHg3SjMJOE0D\/rC\/Yl0opGlaZ+5A71qR5SdZNjf\/Ajg="`
that become:
`"tHg3SjMJOE0D/rC/Yl0opGlaZ+5A71qR5SdZNjf/Ajg="`

Basically, the backslashes are removed from every string in JSON source.

Is this by design? Or is it an unintended behaviour?

**Comment by goatfungus** [comment]
The \ character is typically used as an escape character, indicating to the software to treat the following character differently. This is commonly used to handle special characters like newlines (\n), carriage returns (\r), or quotes (\") within the data, all of which would break the JSON string if written normally.

In any case, / and \/ are both decoded as a single / character by the game and the editor, so it makes no difference which is used. This would not be an issue unless the game's JSON decoder is made more restrictive (which I doubt would ever happen since it's well established), so nothing to worry about.

Lastly, this is not a stripping of ALL backslashes, as they are still used in the editor for other special characters such as carriage returns, newlines, tabs, quotes, etc.
